documentation/builders/webapp/playlists-livestreams-podcasts.md

```diff
@@ -0,0 +1,141 @@
+# Playlists, Livestreams and Podcasts
+
+By default, the Phoniebox represents music based on its metadata like album name, artist or song name. The hierarchy and order of songs is determined by their original definition. If you prefer a specific list of songs to be played, you can use playlists (files ending with `*.m3u`). Phoniebox also supports livestreams and podcasts (if connected to the internet) through playlists.
```

Comment:
Should we call it `Jukebox` or `Phoniebox` ? 
I think in most docs its `Jukebox` (legal actions for the name "Phoniebox" on mind?).

Reply:
At the very beginning of the v2 README it is called `Phoniebox` ;)

Reply:
True 😄 
I did now search in the v3 docs and both is often used, so i was wrong. 

Maybe we should define to only use one wording? `Phoniebox`?

Reply:
I am actually voting for `Jukebox`. Mainly for reasons @AlvinSchiller called out. We could then use `Phoniebox` only in discussions or in a few positions where it makes sense.